app/src/main/res/layout/cards_buttons_layout.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
please, use ConstrainLayout instead of nested LinearLayouts. The current implementation is very inefficient